README.md

```diff
@@ -34,10 +34,12 @@ require('tabline').setup({})
 
 ```lua
 require('tabline').setup({
-    show_index = true,        -- show tab index
-    show_modify = true,       -- show buffer modification indicator
-    modify_indicator = '[+]', -- modify indicator
-    no_name = '[No name]',    -- no name buffer name
+    show_index = true,         -- show tab index
+    show_modify = true,        -- show buffer modification indicator
+    bracket_character = true,  -- show brackets in name
```

Comment:
It's not easy to config both the character, as a result, we may change this to `show_bracket`.